server/lib/src/server/migrations.rs

```diff
@@ -806,6 +806,7 @@ impl<'a> QueryServerWriteTransaction<'a> {
             SCHEMA_ATTR_DENIED_NAME.clone().into(),
             SCHEMA_ATTR_CREDENTIAL_TYPE_MINIMUM.clone().into(),
             SCHEMA_ATTR_WEBAUTHN_ATTESTATION_CA_LIST.clone().into(),
+            SCHEMA_ATTR_APPLICATION_PASSWORD.clone().into(),
```

Comment:
This migration framework is no longer called on upgrades. We need to put this elsewhere, but that can wait for later into the PR since for now in "tests" and on new instances this will work.